docs/UserGuide.md

```diff
@@ -706,7 +708,7 @@ Alias is the command for you. The `alias` command can help you to shorten input
 
 | Input     | Output | Usage of Alias | Output |
 | ----------- | ----------- | ---------- | ------------- |
-| `alias al/MyList cmd/list` | `Added alias myList: [list]` | `myList` | `Listed all study spots` |
+| `alias al/MyList cmd/list` | Added alias myList: [list] | `myList` | `Listed all study spots` |
```

Comment:
* Is there a reason why `Added alias myList: [list]` should not be in codeblocks?
* `myList` should be `MyList` in the output column
* this example may not be the best as we wrote that `alias` command helps to shorten commands but it lengthened the command input in this example...
* Perhaps we should include that alias is case-sensitive?

Reply:
1. Isn't Added alias myList: [list] an output and not a command? I think for the UG all the outputs have no codeblocks. 
2. okay!
3. changed MyList to ml
4. okay!